modules/core/src/v2/coins/xlm.ts

```diff
@@ -382,6 +392,20 @@ export class Xlm extends BaseCoin {
     }
   }
 
+  /**
+   * Return a Stellar Asset in coin:token form (i.e. (t)xlm:<code>-<issuer>)
+   * If the asset is XLM, return the chain
+   * @param {stellar.Asset} asset - instance of Stellar Asset
+   */
+  getTokenFromStellarAsset(asset: stellar.Asset) {
```

Comment:
Please rename to `getTokenNameFromStellarAsset` since this returns a string instead of a `StellarToken`

Reply:
ok, done